README.md

```diff
@@ -9,7 +9,7 @@ An simple application to help you  keep track of the tasks you need to do and ma
 ## Built With
 
 - HTML
-- CSS
+- CSS (Animate CSS Framework inclusive)
 - JavaScript
 - Webpack
 
```

Comment:
- [OPTIONAL] Please kindly add a license section to your readme file as in this [template](https://github.com/microverseinc/readme-template), as this will make your readme file more informative and professional.